src/react-native/javascript.ts

```diff
@@ -63,11 +69,17 @@ export async function addSentryInit({
       'By default, all text content, images, and webviews will be masked for privacy. You can customize this in your code later.',
     );
   }
+  if (enableFeedbackWidget) {
+    clack.log.info(
+      `The Feedback Widget will be enabled with default settings.`,
```

Comment:
We should probably move this message just after enabling the step and do the same for session replay. 
As shown in the screenshot the info texts appear mixed when both flags are turned on and this might be confusing.

<img width="1194" alt="Screenshot 2025-04-18 at 4 38 14 PM" src="https://github.com/user-attachments/assets/2ceb8428-b61e-4dd3-bccf-dfc322ea2bbd" />

Reply:
Updated with 12fac4f090bca07cac6f9db01b39af988315d37b

<img width="1187" alt="Screenshot 2025-04-22 at 1 14 37 PM" src="https://github.com/user-attachments/assets/681ce154-d496-4a54-8bb6-baa45609703e" />